unit2-java/lesson07.rst

```diff
@@ -4,15 +4,19 @@ Unit 2 (Java) - Lesson 7 Learning Objectives
 Goals
 -----
 
+- Understand how polymorphism is enabled by interfaces and inheritance.
+- Understand and use inheritance to create more flexible code. 
+
 Objectives
 ----------
 
-- Define polymorphism
+- Explain polymorphism and how it is useful
 - Describe what an interface is and how it enables polymorphism
-- Explain whether an interface can be instantiated
+- Explain why it is preferable to use interface types in method declarations ("coding to an interface")
+- Explain why an interface can not be instantiated
 - Implement common standard Java interfaces
-- Define what a default method is and whether you should write one when creating a new interface
+- Explain what a default method is and how they may be used with interfaces
 - Use the appropriate interface for collection types depending on the behavior they must exhibit
 - Describe the differences between abstract classes and interfaces
-- Explain what the benefits are to "coding to an interface"
 - Use the ``Comparator.compare()`` method
+- Write unit tests that test interface-based code
```

Comment:
Similar to the unit 6 comment.

"...tests to test..." kinda bothers me. How about something like, "Write unit tests to verify..." or, "Write tests for interface-based code."